README.md

```diff
@@ -3,7 +3,7 @@
   <br>
 </h1>
 
-[![Build Status](https://travis-ci.com/kubeflow/katib.svg?branch=master)](https://travis-ci.com/kubeflow/katib)
+[![Build Status](https://github.com/kubeflow/katib/actions/workflows/test-go.yaml/badge.svg)](https://github.com/kubeflow/katib/actions/workflows/test-go.yaml)
```

Comment:
Should we point to the `master` branch ? Similar to this: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge#using-the-branch-parameter ?

```suggestion
[![Build Status](https://github.com/kubeflow/katib/actions/workflows/test-go.yaml/badge.svg?branch=master)](https://github.com/kubeflow/katib/actions/workflows/test-go.yaml?branch=master)
```